code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -282,7 +282,7 @@
 				return
 			var/obj/item/weapon/weldingtool/WT = W
 			if(WT.remove_fuel(0, user))
-				health = min(maxHealth, health+10)

```

Comment:
Wow who ever thought this was a good idea? I recently fixed a similar bug on humans..